mllib/src/main/scala/org/apache/spark/ml/tuning/CrossValidator.scala

```diff
@@ -102,12 +102,6 @@ class CrossValidator(override val uid: String) extends Estimator[CrossValidatorM
   /** @group setParam */
   def setNumFolds(value: Int): this.type = set(numFolds, value)
 
-  override def validateParams(paramMap: ParamMap): Unit = {
```

Comment:
Sorry, I should have been clearer: CrossValidator still needs to call validateParams() for each of the estimatorParamMaps.  It should copy the estimator with each paramMap, and then call validateParams for that copy.